### ISSUE TITLE
(CompileError) Module NimbleParsec is not loaded and could not be found

### ISSUE THREAD

**Issue by rubynho** [kipcole9/money#114]
Hello,

I'm having trouble compiling the dependency on my Elixir project. I followed the installation guide in the README and after install when I open the console (iex -S mix) I get this error:
```
==> ex_money
Compiling 21 files (.ex)

== Compilation error in file lib/money/parser.ex ==
** (CompileError) lib/money/parser.ex:4: module NimbleParsec is not loaded and could not be found
```
I'm using the `Elixir v1.9.4` and `Erlang/OTP v22`.

Thanks in advance!

**Comment by coladarci** [comment]
Can you check your lock file for which version of money is being used?

**Comment by rubynho** [comment]
Here is my lock file:
```diff
 %{
   "bunt": {:hex, :bunt, "0.2.0", "951c6e801e8b1d2cbe58ebbd3e616a869061ddadcc4863d0a2182541acae9a38", [:mix], [], "hexpm"},
+  "cldr_utils": {:hex, :cldr_utils, "2.8.0", "a1355a658fdf7118a678002a5333562d464e1bbdc3c89a5e0c3d8088038e57f5", [:mix], [{:decimal, "~
   "credo": {:hex, :credo, "1.2.2", "f57faf60e0a12b0ba9fd4bad07966057fde162b33496c509b95b027993494aab", [:mix], [{:bunt, "~> 0.2.0", [he
+  "decimal": {:hex, :decimal, "1.8.1", "a4ef3f5f3428bdbc0d35374029ffcf4ede8533536fa79896dd450168d9acdf3c", [:mix], [], "hexpm"},
+  "ex_cldr": {:hex, :ex_cldr, "2.13.0", "742f14a4afcfea61a190d603d8e555d2c91d71e4e8fc2520d5dc35616969e225", [:mix], [{:cldr_utils, "~> 
+  "ex_cldr_currencies": {:hex, :ex_cldr_currencies, "2.4.1", "a8e8330a6d0712b8bb34c5e3759311da1d53fa8bebef163d72615f0ea60c0738", [:mix]
+  "ex_cldr_numbers": {:hex, :ex_cldr_numbers, "2.12.0", "2c38265d680e29d3440f19b63a5054b94d41f9006cad3e82cd695affe7997035", [:mix], [{:
+  "ex_money": {:hex, :ex_money, "4.4.2", "7c35e05eda5efeedf1bbe263bf097b9c240e02f73c5d56e1bd69467f3e625e5b", [:mix], [{:decimal, "~> 1.
   "jason": {:hex, :jason, "1.1.2", "b03dedea67a99223a2eaf9f1264ce37154564de899fd3d8b9a21b1a6fd64afe7", [:mix], [{:decimal, "~> 1.0", [h
 }

```

**Comment by rubynho** [comment]
I could get it working after updating the dependency to v5.0.1!

**Comment by coladarci** [comment]
Yup that was going to be my suggestion; did you also bump elixir to 1.10?

**Comment by rubynho** [comment]
Not yet but I will soon!

**Comment by kipcole9** [comment]
Sorry for the inconvenience. Using the 5.x release series is definitely preferred but I have also now published [ex_money version 4.4.4](https://hex.pm/packages/ex_money/4.4.3) that makes `nimble_parsec` a required dependency (not optional) to make is easier for current users to upgrade.

Thanks for the report.
